src/main/java/net/glowstone/io/entity/LivingEntityStore.java

```diff
@@ -148,6 +130,68 @@ public void load(T entity, CompoundTag compound) {
         }
     }
 
+    private void loadEquipment(T entity, EntityEquipment equip, CompoundTag compound) {
+        // Deprecated since 15w31a, left here for compatibilty for now
+        if (compound.isList("Equipment", TagType.COMPOUND)) {
+            List<CompoundTag> list = compound.getCompoundList("Equipment");
+            if (list.size() == 5) {
```

Comment:
I can't see a Situation where:
- A world is saved with an incomplete list of "Equipment", "HandItems" or "ArmorItems"
- and the client still working

Just tried this situation out. Client is just black, and I can't move.

Since it is not safe to have incomplete lists in vanilla, I don't see how useful handling of this is going to be for us.